synfig-studio/src/gui/app.cpp

```diff
@@ -3773,10 +3774,26 @@ App::dialog_paragraph(const std::string &title, const std::string &message,std::
 
 	dialog.get_content_area()->pack_start(text_view);
 
-	dialog.add_button(_("Cancel"), Gtk::RESPONSE_CANCEL)->set_image_from_icon_name("gtk-cancel", Gtk::ICON_SIZE_BUTTON);
-	dialog.add_button(_("OK"),   Gtk::RESPONSE_OK)->set_image_from_icon_name("gtk-ok", Gtk::ICON_SIZE_BUTTON);
+	dialog.add_button(_("_Cancel"), Gtk::RESPONSE_CANCEL)->set_image_from_icon_name("gtk-cancel", Gtk::ICON_SIZE_BUTTON);
+	dialog.add_button(_("_OK"),   Gtk::RESPONSE_OK)->set_image_from_icon_name("gtk-ok", Gtk::ICON_SIZE_BUTTON);
 	dialog.set_default_response(Gtk::RESPONSE_OK);
 
+	text_view.signal_key_press_event().connect(( [&](GdkEventKey *ev) {
+
+		SYNFIG_EXCEPTION_GUARD_BEGIN()
+		if (	(ev->type == GDK_KEY_PRESS) &&
+				(ev->keyval == (GDK_KEY_Return) || ev->keyval == (GDK_KEY_KP_Enter) ) &&
+				(ev->state == GDK_CONTROL_MASK) )
+			{
+			dialog.response(Gtk::RESPONSE_OK);
+			return true;
+			}
+			return false;
```

Comment:
Indentation is a bit weird here :)

Reply:
yea, I admit it did look a bit weirx xd. I fixed it up abit and indented the  "if statement" in the way you guys proposed in the forums. I am guilty of using a space or two though to fix the allignment of the multi-lined condition but I think its fine. if not please let me know 😊.